zcash_primitives/src/sapling/note_encryption.rs

```diff
@@ -649,8 +650,8 @@ mod tests {
             MemoBytes::empty(),
             &mut rng,
         );
-        let epk = *ne.epk();
-        let ock = prf_ock(&ovk, &cv, &cmu.to_repr(), &epk_bytes(&epk));
+        let epk = EphemeralPublicKey::from_inner(*ne.epk());
+        let ock = prf_ock(&ovk, &cv, &cmu.to_repr(), &epk.to_bytes());
```

Comment:
Clippy complaining on line 660 below after these changes.

Reply:
I'll fix clippy issues in a follow-up PR once the dust settles.